src/components/views/OpptjeningView/OpptjeningView.js

```diff
@@ -66,8 +69,24 @@ export const OpptjeningView = () => {
         }
     };
 
+    const Veiledertext = () => {
```

Comment:
Nesten så den kunne vært en egen component i en separat fil menmen nitpick.

Reply:
Ja - vurderte det - men droppet det for å slippe å skrive egne tester og doc.  Men er egentlig enig i at det burde være egen komponent.

Reply:
Har laget en ny komponent

Reply:
Ser nå at du mente at Veiledertext skulle være en egen komponent, mens jeg har laget hele veilederen til en egenkomponent som tar inn teksten......

Reply:
Er det slik at hver komponent krever doc og test?

Reply:
Krever og krever - men har forsøkt å følge det mønsteret ja.